utilities/imu.py

```diff
@@ -0,0 +1,37 @@
+from wpilib.interfaces import PIDSource
+from utilities.navx import NavX
+from utilities.bno055 import BNO055
+
+
+class IMU:
```

Comment:
I'm not sure I'm a fan of doing it this way. Given that our control loop is too slow, shouldn't we be trying to remove overhead, not introducing more?

Reply:
How much overhead could wrapping the getAngle calls in one function introduce really, though?